src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+//TODO: Decide about i18n solution or simply a JSON file where we can load translations.
+
+const TRANSLATIONS = {
+  SHIP_TO_ME: 'Ship to Me',
+  ARRIVES: 'Arrives'
+};
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+Container.displayName = 'Container';
```

Comment:
We are no longer using enzyme so it is not needed anymore I guess.

Reply:
@blurbyte I didn't find better way to find this element during testing?

Reply:
Not really sure why you are testing onClick here. In future if you want to target component directly set **data-testid**. Also I added **react-testing-library** for interaction tests (maybe we event don't need RTR anymore, have to check it alter).